core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1290,7 +1291,7 @@ class DAGScheduler(
       case TaskResultLost =>
         // Do nothing here; the TaskScheduler handles these failures and resubmits the task.
 
-      case other =>
+      case _: ExecutorLostFailure | TaskKilled | UnknownReason =>
```

Comment:
If so, I think the compiler will report uncompleted matching as a warning and the build will fail. Right?